src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -441,4 +445,59 @@ public static void DuplexCallback_Throws_FaultException_ReturnsFaultedTask()
             }
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    [ActiveIssue(485)]
+    public static void TcpClientCredentialType_Certifiacte_CustomValidatorFail_EchoString()
+    {
+        string expectedExceptionMessage = "[MyX509CertificateValidator] Certificate was not issued by a trusted issuer";
+        string clientCertThumb = null;
+        EndpointAddress endpointAddress = null;
+        string testString = "Hello";
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
+
+        try
+        {
+            var exception = Assert.Throws<Exception>(() =>
+            {
+                // *** SETUP *** \\
+                NetTcpBinding binding = new NetTcpBinding(SecurityMode.Transport);
+                binding.Security.Transport.ClientCredentialType = TcpClientCredentialType.Certificate;
+
+                endpointAddress = new EndpointAddress(new Uri(Endpoints.Tcp_ClientCredentialType_Certificate_CustomValidation_Address),
+                    new DnsEndpointIdentity(Endpoints.Tcp_VerifyDNS_HostName));
+                clientCertThumb = BridgeClientCertificateManager.LocalCertThumbprint; // ClientCert as given by the Bridge
+
+                factory = new ChannelFactory<IWcfService>(binding, endpointAddress);
+                factory.Credentials.ServiceCertificate.Authentication.CertificateValidationMode = X509CertificateValidationMode.None;
```

Comment:
Thanks Hong, when I moved the test from my desktop implementation to OSS I managed to lose that setting, thanks for catching it.
I'll remove the negative test since you already got it.